tools/deployment/macos_packaging/pkg/productbuild.sh

```diff
@@ -92,7 +92,7 @@ function main() {
   parse_args $@
 
   # Overwrite the package identifier.
-  PKG_IDENTIFIER="com.facebook.osquery"
+  PKG_IDENTIFIER="com.osquery.osquery"
```

Comment:
When I made the apple profile, I made it as `io.osquery.agent`, mostly because the doubled osquery wasn't aesthetically pleasing to me. I suspect we should be consistent. I think I can regenerate the apple profile, or we should swap these to that.